### PR TITLE
Add posing test code coverage reports of nighly builds

### PR DESCRIPTION
**Pull request by bdolewski-intellias**

### CODE REVIEW

File: .github/workflows/pod-tests-html.yml [hunk 1]
```diff
@@ -108,6 +108,116 @@ jobs:
       - name: Check to see if build failed
         run: |
           ! grep -o "Testing cancelled because the build failed." errors.log || exit 1
+      - name: Generate tests and code coverage reports
+        if: always()
+        continue-on-error: true
+        run: |
+          cd ~/LocalCheckouts/${{ github.event.repository.name }}
+          pwd
```
Comment (salmin-skelia) [inline .github/workflows/pod-tests-html.yml:116]: I fell like this `pwd` is used for debug, remove pls

**Review by salmin-skelia — CHANGES_REQUESTED** [review]
Remove all debug prints pls

File: .github/workflows/pod-tests-html.yml [hunk 2]
```diff
@@ -108,6 +108,116 @@ jobs:
...
+          rm -rf XcresultReportPackages
+          mkdir XcresultReportPackages
+          cd XcresultReport
+          xcodebuild -scheme xcresult-report -destination arch=x86_64 -derivedDataPath ./DerivedData/XcresultReport -clonedSourcePackagesDirPath ../XcresultReportPackages
+          cp ./DerivedData/XcresultReport/Build/Products/Debug/xcresult-report ./
+          xcresultReportCmd="./xcresult-report --xcresult-filepath ~/LocalCheckouts/${{ github.event.repository.name }}/test-results.xcresult"
+          eval $xcresultReportCmd
+          pwd
+          ls -lat
+          echo "testSummaryReport.html"
```
[the remark this answers is not in the document]
Comment (bdolewski-intellias) [reply .github/workflows/pod-tests-html.yml:137]: I think that was for debugging purposes. I'll get rid of it in next step which is to convert this whole part as "composite action" :) 
Now I'd like to see if I can post reports in right channel. This whole diff is a copy-paste from another YAML file making even more duplication, I know :/

**Comment by bdolewski-intellias** [comment]
> Remove all debug prints pls

Yeah, I'll get rid of those debug stuff. It clutters the script anyway.

**Comment by bdolewski-intellias** [comment]
@salmin-skelia debug stuff removed

**Review by salmin-skelia — APPROVED** [review]

**Pull request merged**